### PR TITLE
Make Generated tests work

### PR DESCRIPTION
**Pull request by cazfletch**
Updated generated tests
Updated generator version to create v2 contracts

contributes to #1778 and #1250

Signed-off-by: Caroline Fletcher <cazfletch@gmail.com>

### CODE REVIEW

File: packages/blockchain-extension/extension/commands/testSmartContractCommand.ts [hunk 1]
```diff
@@ -535,7 +535,7 @@ async function updateBuildFile(buildFile: string, state: any): Promise<void> {
             'fabric-gateway-java': {
                 groupId: 'org.hyperledger.fabric',
                 artifactId: 'fabric-gateway-java',
-                version: '1.4.2',
+                version: '2.1.0',
```
Comment (Jakeeyturner) [inline packages/blockchain-extension/extension/commands/testSmartContractCommand.ts:538]: It wasn't previously, but should `version: FABRIC_NETWORK_VERSION`?

File: packages/blockchain-extension/templates/jsTestSmartContractTemplate.ejs [hunk 2]
```diff
@@ -29,7 +29,7 @@ const path = require('path');
 <%_ }-%>
 
 <%_ if (contractName !== '') { _%>
-describe('<%=contractName%>-<%=chaincodeLabel%>' , () => {
+describe('<%=contractName%>-<%=chaincodeLabel%>' , async () => {
```
Comment (Jakeeyturner) [inline packages/blockchain-extension/templates/jsTestSmartContractTemplate.ejs:32]: Why does this need to be `async`?

File: packages/blockchain-extension/extension/commands/testSmartContractCommand.ts (hunk 1 above)
Comment (cazfletch) [reply packages/blockchain-extension/extension/commands/testSmartContractCommand.ts:538]: good point but i think we need separate variables for java and node

File: packages/blockchain-extension/templates/jsTestSmartContractTemplate.ejs (hunk 2 above)
Comment (cazfletch) [reply packages/blockchain-extension/templates/jsTestSmartContractTemplate.ejs:32]: it doesn't lol

**Pull request merged**